ItemLibrary/Smartphone.cs

```diff
@@ -6,8 +6,8 @@ namespace ItemLibrary
 {
     public class Smartphone : Item
     {
-        public string FrontCamera { get; set; }
-        public string EndCamera { get; set; }
+        public List<int> FrontCamera { get; set; }
```

Comment:
Better field name would be great. Cause camera can have megapixels (MP), diafragm (F/number) or else

Reply:
renamed fields